contrib/packs/actions/install.meta.yaml

```diff
@@ -32,3 +32,8 @@
       description: "Use Python 3 binary when creating a virtualenv for this pack."
       required: false
       default: false
+    skip:
+      type: "boolean"
+      description: "Set to True to skip pack dependency installations."
```

Comment:
`st2 pack install abc --skip` sounds as too generic flag. It's not clear how _skip_ affects _pack install_.

Something like `--skip_dependencies`, `--skip_deps` or any other similar more explicit analog would work better.

Reply:
Here is even better example:

```
$ pip install --help | grep deps
  --no-deps                   Don't install package dependencies.
```

Reply:
Like `--no-deps` and will change to it.